spec/order_spec.rb

```diff
@@ -0,0 +1,64 @@
+describe Order do
+  let(:menu) do
+    double :menu,
+    show: [{dish: 'BBQ',       price: 10},
+           {dish: 'Sushi',     price: 6}]
+  end
+
+  it "has customer choose a dish and quantities" do
+    expect(subject.choose_dish(menu, 'BBQ', 3))
+      .to eq([{dish: 'BBQ', price: 10, quantities: 3}])
+  end
+
+  it "accepts more than one dish" do
+    subject.choose_dish(menu, 'BBQ', 3)
+    subject.choose_dish(menu, 'Sushi', 3)
+    expect(subject.orders).to eq([{dish: 'BBQ', price: 10, quantities: 3},
+                                 {dish: 'Sushi', price: 6, quantities: 3}])
+  end
+
+  it "shows the total amount" do
+    subject.choose_dish(menu, 'BBQ', 3)
+    subject.choose_dish(menu, 'Sushi', 3)
+    expect(subject.checkout)
+      .to eq("BBQ: 3 * 10 = 30 | Sushi: 3 * 6 = 18 | The total amount is £48")
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)